src/ecmascript_simd.js

```diff
@@ -538,18 +515,40 @@ if (typeof SIMD.int32x4 === "undefined") {
     get: function() { return tobool(this.w); }
   });
 
+}
+
+if (typeof SIMD.int32x4.allTrue === "undefined") {
+  /**
+    * Check if all 4 lanes hold a true value (bit 31 == 1)
+    * @param {int32x4} v An instance of int32x4.
+    * @return {Boolean} All 4 lanes holds a true value
+    */
+  SIMD.int32x4.allTrue = function(v) {
+    if (!(v instanceof SIMD.int32x4)) {
+      throw new TypeError("argument is not a int32x4.");
+    }
+    return ((v.x & 0x80000000) !== 0) &&
+           ((v.y & 0x80000000) !== 0) &&
+           ((v.z & 0x80000000) !== 0) &&
+           ((v.w & 0x80000000) !== 0);
```

Comment:
I take that back!  Turns out that the internal rep for SIMD values are now back to typed arrays instead of individual properties, so yes, I should use tobool().  How about renaming it to toBool(), BTW